[2025-10-02T14:20:13.460+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T14:20:13.476+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.run_write_postgres manual__2025-10-02T14:20:05.024652+00:00 [queued]>
[2025-10-02T14:20:13.484+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.run_write_postgres manual__2025-10-02T14:20:05.024652+00:00 [queued]>
[2025-10-02T14:20:13.485+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-02T14:20:13.507+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_write_postgres> on 2025-10-02 14:20:05.024652+00:00
[2025-10-02T14:20:13.512+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=14210) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T14:20:13.514+0000] {standard_task_runner.py:63} INFO - Started process 14212 to run task
[2025-10-02T14:20:13.514+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'main', 'run_write_postgres', 'manual__2025-10-02T14:20:05.024652+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpufmfxy5i']
[2025-10-02T14:20:13.515+0000] {standard_task_runner.py:91} INFO - Job 120: Subtask run_write_postgres
[2025-10-02T14:20:13.550+0000] {task_command.py:426} INFO - Running <TaskInstance: main.run_write_postgres manual__2025-10-02T14:20:05.024652+00:00 [running]> on host 5adecdfd8c83
[2025-10-02T14:20:13.612+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='run_write_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T14:20:05.024652+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-02T14:20:05.024652+00:00'
[2025-10-02T14:20:13.614+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T14:20:17.750+0000] {logging_mixin.py:188} INFO - Получен валидный DataFrame
[2025-10-02T14:20:17.753+0000] {logging_mixin.py:188} INFO - Empty DataFrame
Columns: []
Index: []
[2025-10-02T14:20:17.753+0000] {logging_mixin.py:188} INFO - Размер: (0, 0)
[2025-10-02T14:20:17.754+0000] {logging_mixin.py:188} INFO - Вот сам DataFrame
[2025-10-02T14:20:17.755+0000] {logging_mixin.py:188} INFO - Empty DataFrame
Columns: []
Index: []
[2025-10-02T14:20:19.795+0000] {logging_mixin.py:188} INFO - +-----+-----+
| Name|Value|
+-----+-----+
|Alice|    1|
|  Bob|    2|
|Cathy|    3|
+-----+-----+
[2025-10-02T14:20:20.755+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-02T14:20:20.756+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T14:20:20.763+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=main, task_id=run_write_postgres, run_id=manual__2025-10-02T14:20:05.024652+00:00, execution_date=20251002T142005, start_date=20251002T142013, end_date=20251002T142020
[2025-10-02T14:20:20.785+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T14:20:20.800+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-02T14:20:20.804+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
